README.md

```diff
@@ -15,11 +19,24 @@ This repository contains example notebooks and documentation for working with [E
 
 ## Getting started
 
-### Setup Github integration in Enterprise Terra
-
-Go to [Enterprise Terra](https://verily.com/solutions/terra/) and:
-1. Add your SSH key to GitHub. 
-2. Create a personal workspace where you can test out many examples, such as this one. Name it something like *"Personal workspace {your email address} {YYYYMMDD}"*.
-3. [Optional] If you want Terra to clone this repo automatically, add it as a reference in your workspace. Otherwise you will need to clone it manually, which is pretty easy.
-4. Create a Cloud Environment for yourself. Name it something like *"{your email address}-{YYYYMMDD}"*.
-5. Be sure to run notebook [workspace_setup.ipynb](https://github.com/DataBiosphere/terra-axon-examples/blob/main/workspace_setup.ipynb) first before running any other notebook files. It sets up some buckets and a BigQuery dataset that all the other notebooks refer to by reference and expect to exist.
+### Set up Github integration in Enterprise Terra
+
+1. Visit [Enterprise Terra](https://verily.com/solutions/terra/)
+1. [Add your SSH key](https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/) to GitHub.
+
+### Configure your workspace and cloud environments
+
+1. Create a personal workspace where you can run examples, including the ones in this repo. Name it something like *"Personal workspace {your email address} {YYYYMMDD}"*.
+1. [Optional] If you want Terra to clone this repo automatically, [add it as a reference in your
+   workspace](https://terra-docs.api.verily.com/docs/how_to_guides/add_repo_to_ws/). Otherwise, you
+   can clone it manually, which is straightfoward.
```

Comment:
Typo in `straightforward`.  Might want to be explicit on how to clone the repo, just in case.